yoink-firmware

```diff
@@ -8,7 +8,16 @@ fi
 
 # Copy over firmware files from windows installation
 mkdir /windows
-mount /dev/disk/by-label/Windows /windows -t ntfs3 -r
+type="$(command -v mount.ntfs3)"
+if [ -z "$type" ]; then
+	type="$(command -v mount.ntfs)"
+fi
+if [ -z "$type" ]; then
+	echo "Can't mount NTFS partitions!"
+	exit 2
+fi
+
+mount /dev/disk/by-label/Windows /windows -t "$type" -r 
```

Comment:
idk this whole thing looks sus, i can't find references tot `mount.ntfs3`, `mount.ntfs /dev/disk/by-label/Windows /windows -r` would probably work fine?

Reply:
```
[macc24@mayophus ~]$ mount.ntfs --help

ntfs-3g 2022.10.3 external FUSE 29 - Third Generation NTFS Driver
		Configuration type 7, XATTRS are on, POSIX ACLS are on

Copyright (C) 2005-2007 Yura Pakhuchiy
Copyright (C) 2006-2009 Szabolcs Szakacsits
Copyright (C) 2007-2022 Jean-Pierre Andre
Copyright (C) 2009-2020 Erik Larsson

Usage:    ntfs-3g [-o option[,...]] <device|image_file> <mount_point>

Options:  ro (read-only mount), windows_names, uid=, gid=,
          umask=, fmask=, dmask=, streams_interface=.
          Please see the details in the manual (type: man ntfs-3g).

Example: ntfs-3g /dev/sda1 /mnt/windows

Plugin path: /usr/lib/ntfs-3g

News, support and information:  https://github.com/tuxera/ntfs-3g/
```
it wouldn't

Reply:
maybe if it tries mounting with `mount -t ntfs`, then `mount -t ntfs`, then `mount.ntfs-3g` lol